TODOKit/Intents.intentdefinition

```diff
@@ -325,7 +325,7 @@
 	<key>INIntentDefinitionNamespace</key>
 	<string>WyMQQ2</string>
 	<key>INIntentDefinitionSystemVersion</key>
-	<string>19E287</string>
+	<string>18G103</string>
```

Comment:
what are these changes? does it happen automatically?

Reply:
This change is for donating the shortcut.